bitsandbytes/__init__.py

```diff
@@ -14,13 +15,22 @@
 )
 from .cextension import lib
 from .nn import modules
+from .backends import register_backend
 
 if lib and lib.compiled_with_cuda:
-    from .backends import register_backend
     from .backends.cuda import CUDABackend
     from .optim import adam
 
     register_backend("cuda", CUDABackend())
+
+elif torch.xpu.is_available():
```

Comment:
If the torch version is cpu, then torch.xpu will have attr error like `module 'torch' has no attribute 'xpu'`

Reply:
There's some overlap here with the PR that I have #1173. On that one it looks like this:

```python
# Register Intel XPU backend, if available.
if hasattr(torch, "xpu") and torch.xpu.is_available():
    from .backends.xpu import XPUBackend

    register_backend("xpu", XPUBackend())
```

Reply:
@matthewdouglas Thanks for the info! So, maybe I need to wait for your PR landed first. Anyway, we can still work on this PR and rebase then.

Reply:
XPU backend removed from this PR